sensing/src/main/java/com/google/android/sensing/impl/SensorManagerImpl.kt

```diff
@@ -170,30 +180,29 @@ internal class SensorManagerImpl(context: Context, private val database: Databas
                     resourceTitle = outputTitle,
                     contentType = outputFormat,
                     localLocation = File(context.filesDir, outputFolder).absolutePath,
-                    /** TODO Update remoteLocation for Sensing1.0. */
-                    remoteLocation = "",
+                    remoteLocation = toUploadFullUrl,
                     status = RequestStatus.PENDING
                   )
                 )
                 try {
                   database.getCaptureInfo(this.captureId).let { captureInfo ->
-                  /**
-                   * Back to original coroutine context. For application this will generally run in
-                   * the main thread.
-                   */
-                  val processedString =
-                      withContext(lifecycleOwner.lifecycleScope.coroutineContext) {
-                        it.listener?.onComplete(captureInfo)
-                        /** TODO PostProcess. */
-                      }
+                    /**
+                     * Back to original coroutine context. For application this will generally run
+                     * in the main thread.
+                     */
+                    withContext(lifecycleOwner.lifecycleScope.coroutineContext) {
+                      it.listener?.onComplete(captureInfo)
+                      /** TODO PostProcess. */
+                    }
+                    /** TODO move this to a default PostProcessor. */
+                    createUploadRequest(captureInfo, it.captureRequest!!)
                   }
-                } catch (e: ResourceNotFoundException) {
+                } catch (e: Exception) {
```

Comment:
why this change? this is worse than before as it's less specific.

Reply:
Apart from ResourceNotFoundException, the createUploadRequest may throw other exceptions like IOException (when zipping) or Transaction error when inserting UploadRequest into the db.
So I thought of at least catching all if not the specific ones.
Can later shift to the specific exceptions when any application asks for it.